src/__tests__/__snapshots__/Banner.js.snap

```diff
@@ -118,6 +118,138 @@ exports[`Banner renders with blue bg 1`] = `
       height={24}
       mr={3}
       mt="-2px"
+      theme={
```

Comment:
Didn't think about it before, but this definitely makes the snapshots a lot bigger 😅